snippets/csharp/VS_Snippets_CLR/conceptual.basicio.textfiles/cs/append.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.IO;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        AppendText();
```

Comment:
There's no need here to put the code in a separate AppendText method that's called from Main; all of the code can just go in Main. That's true of the next examples as well.